samples/FXL_I2C/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
```

Comment:
This file is used for the testing framework, will have to check when doing the zephyrproject PR if this is okay. For now it's okay like this